hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -442,8 +443,21 @@ public static String addOrIncrementIndexInName(String name) {
         return name + '-' + index;
     }
 
-    public static String sanitizeLoggerNamePart(String name) {
-        return name.replace('.', '_');
+    public static ILogger prefixedLogger(ILogger logger, String prefix) {
+        return new PrefixedLogger(logger, prefix);
+    }
+
+    public static String prefix(String jobName, long jobId, String vertexName) {
```

Comment:
The name is too generic for `Util`. Use `loggerPrefix` at least. Or maybe move them to the `PrefixedLogger`